src/main/java/mate/controller/car/CreatingCarController.java

```diff
@@ -0,0 +1,36 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/cars/creatingCar")
```

Comment:
make sure to check out [this ](https://mate-academy.github.io/jv-program-common-mistakes/java-web/web-practice/java-web-practice)